IFC4x3/Properties/l/LockingType_1vFkVHKPjAvvxUxFl0WWv7/DocProperty.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="LockingType_1vFkVHKPjAvvxUxFl0WWv7" Name="LockingType" UniqueId="793ee7d1-519b-4ae7-9ede-ecfbc0820e47" PropertyType="p_enumeratedvalue">
```

Comment:
We are missing a primary data type IfcLabel.

Reply:
I don't believe IfcDoc enforced explicit serialization of an "IfcLabel" datatype on a property template, this is the default.
You can see that here, https://github.com/buildingSMART/IfcDoc/blob/0a884dd67a2d0025952467b956f34b0b2b5cae74/Formats/IfcDoc.Formats/FormatHTM.cs#L2527

Although I see that isn't happening when publishing properties to ifcxml etc.  Is it desired to have it explicit?
There is an impact (not measured how significant) on the serialization size and efficiency if all attributes are explicit.  There are other aspects that "default" (I need to check to find other examples).